frontend/src/components/profile/payments.js

```diff
@@ -1,35 +1,31 @@
 import React, { useEffect, useState } from 'react'
-
+import { listTasks, filterTasks } from './frontend/src/actions/taskAction.js'
```

Comment:
Wrong import here @Anshgrover23 

```suggestion
import { listTasks, filterTasks } from '../../actions/taskActions'
```